k8s/secrets.tf

```diff
@@ -31,6 +31,7 @@ resource "kubernetes_secret_v1" "lotus_wallaby_archive_node_secret" {
   data = {
     privatekey = lookup(jsondecode(data.aws_secretsmanager_secret_version.wallaby_archive_lotus[0].secret_string), "private_key", null)
     token      = lookup(jsondecode(data.aws_secretsmanager_secret_version.wallaby_archive_lotus[0].secret_string), "jwt_token", null)
+    nodeid     = lookup(jsondecode(data.aws_secretsmanager_secret_version.wallaby_archive_lotus[0].secret_string), "bootstrap_node_id", null)
```

Comment:
Interesting idea to store it as item in already existed secret :+1:

Reply:
it is just key:value
and we used it for space00 on the same way

we can modify that logic, but you need to change a helm chart logic